btc-testing-endpoints/build.gradle

```diff
@@ -0,0 +1,37 @@
+buildscript {
+    repositories {
+        // gradle plugins repository
+        gradlePluginPortal()
+    }
+}
+
+plugins {
+    id 'jp.co.soramitsu.sora-plugin' version '0.1.2'
+}
+
+dependencies {
+    compile project(":btc")
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+}
+
+apply plugin: 'application'
+apply plugin: 'kotlin'
+apply plugin: 'com.github.johnrengelman.shadow'
+
+mainClassName = "com.d3.btc.endpoints.TestingEndpointsMain"
+shadowDistZip.classifier = 'shadow'
+shadowDistTar.classifier = 'shadow'
+// name of jar file that the shadowJar plugin generates
+shadowJar.archiveName = 'app.jar'
+
+// sora-plugin configs
+soramitsu {
```

Comment:
Do we need to publish testing endpoint on nexsus?

Reply:
I think yes since it should be deployed using deploy scripts (docker compose etc) on testing environments

Reply:
We can change projectGroup = 'd3-deploy' to 'd3-testing' for test images.